src/Employee.java

```diff
@@ -1,14 +1,139 @@
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Scanner;
 
+/**
+ * This class implements a simple employee
+ * 
+ * @author geanina.finaru
+ * @version 1.0
+ */
 public class Employee {
 
 	private String name;
```

Comment:
add some validation here

Reply:
I added the validation for this attribute.